src/databricks/labs/ucx/install.py

```diff
@@ -193,16 +202,22 @@ def run(self):
         workspace_installation.run()
 
     def configure(self) -> WorkspaceConfig:
-        try:
-            return self._installation.load(WorkspaceConfig)
-        except NotFound as err:
-            logger.debug(f"Cannot find previous installation: {err}")
```

Comment:
This is a regression, because we're now enabling lineage every installation. The logic has to remain: if there's an installation, we return config

Reply:
The code will return existing installation config if there is no additional FORCE_INSTALL override right? We have to check that even for existing installations. I will refactor accordingly.